plugins/tiddlywiki/tiddlyweb/tiddlywebadaptor.js

```diff
@@ -76,6 +76,14 @@ TiddlyWebAdaptor.prototype.getStatus = function(callback) {
 			if(err) {
 				return callback(err);
 			}
+
+			//If Browser-Storage plugin is present, add pre-loaded tiddlers to the event queue to sync to the server 
+			if($tw.wiki.getTiddlerText("$:/config/BrowserStorage/Enabled") === "yes" && $tw.boot.preloadDirty != null) {
```

Comment:
With the changes proposed above, this test might be `if($tw.browserStorage && $tw.browserStorage.isEnabled())`